AWSAppSyncClient/AWSAppSyncHTTPNetworkTransport.swift

```diff
@@ -18,6 +18,9 @@ public class AWSAppSyncHTTPNetworkTransport: AWSNetworkTransport {
     var oidcAuthProvider: AWSOIDCAuthProvider? = nil
     var endpoint: AWSEndpoint? = nil
     let authType: AWSAppSyncAuthType
+    // We use the default strategy as exponential retry if someone is using AWSAppSyncHTTPTransport directly.
+    // Currently, retry strategy cannot be changed individually for HTTP network; it is controlled at client level.
+    var retryStrategy: AWSAppSyncRetryStrategy = .exponential
```

Comment:
Add this as an initialization parameter with a default

Reply:
will update.